fastmri/data/mri_data.py

```diff
@@ -194,6 +197,10 @@ def __init__(
             num_examples = round(len(self.examples) * sample_rate)
             self.examples = self.examples[:num_examples]
 
+        if num_cols:
+            self.examples = [
+                ex for ex in self.examples if ex[2]["encoding_size"][1] in num_cols]
```

Comment:
Nit: Could you format this with `black`?

Reply:
Ran `black` on the latest commit.